src/main/webapp/app/core/course/shared/course-sidebar/course-sidebar.component.scss

```diff
@@ -178,3 +178,15 @@ a:not(.btn):not(.tab-link):hover {
         display: none; /* Hide the sidebar when printing */
     }
 }
+// Without ng-deep we fail to override the styles from the tooltip
+::ng-deep .double-arrow-tooltip {
+    left: 2rem !important;
+    margin-bottom: 0.25rem !important;
+}
+::ng-deep .double-arrow-tooltip-no-bottom-items {
+    left: 5.75rem !important;
+    margin-bottom: 0.25rem !important;
+}
+::ng-deep .double-arrow-tooltip-collapsed {
+    left: -2.25rem !important;
+}
```

Comment:
Could we add some more documentation how we got to those values here? 

Such solutions are likely to break every now and then, so it would be great if you can directly infer how they can be updated